tests/unit/lms/services/canvas_api/client_test.py

```diff
@@ -12,12 +12,11 @@
 from lms.services.canvas_api.client import CanvasAPIClient
 from tests import factories
 
+pytestmark = pytest.mark.usefixtures("http_session", "oauth_token")
 
-class TestCanvasAPIClientGetToken:
-    # This is the only test where we fake out the underlying class, because
-    # this _one_ call is just a pass through.
 
-    def test_get_token(self, canvas_api_client, authenticated_client):
+class TestGetToken:
+    def test_it(self, canvas_api_client, authenticated_client):
```

Comment:
There's no point in having a test called `TestGetToken.test_get_token()`, our style is to just call it `TestGetToken.test_it()` in these cases